### PR TITLE
Test coverage improvement

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -151,26 +151,6 @@ subprojects {
         }
     }
 
-    /* Uncomment this block if you need to display console output during the Gradle build.
```
Comment (alexander-yevsyukov) [inline build.gradle]: @alexander-aleksandrov, why did you remove it?

File: build.gradle [hunk 2]
```diff
@@ -184,29 +164,6 @@ subprojects {
         protoc {
             artifact = protobufDependency
         }
-
```
Comment (alexander-yevsyukov) [inline build.gradle]: @alexander-aleksandrov, why did you remove it?

File: build.gradle [hunk 3]
```diff
@@ -33,7 +33,7 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
 
-        spineVersion = '0.8.1-SNAPSHOT'
+        spineVersion = '0.8.2-SNAPSHOT'
```
Comment (alexander-yevsyukov) [inline build.gradle]: Please advance the version as compared with the previous PR.

File: build.gradle (hunk 1 above)
Comment (alexander-aleksandrov) [inline build.gradle]: Fixed

File: build.gradle (hunk 2 above)
Comment (alexander-aleksandrov) [inline build.gradle]: Fixed

File: build.gradle (hunk 3 above)
Comment (alexander-aleksandrov) [inline build.gradle]: Fixed
